reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md

```diff
@@ -0,0 +1,133 @@
+|* create virtual server instance as only management access point to the landscape|Bastion host VPC instance|Create Linux VPC instance that acts as bastion host. Preconfigure ACL and security group rules to allow SSH connectivity (port 22). Add public IP address to the VPC instance. Allow connectivity from a restricted and limited number of public IP addresses. Allow connectivity from IP addresses of the Schematics engine nodes||
+|* create virtual server instance to host basic management services like DNS, NTP, NFS|Management services VPC instance|CreateLinux VPC instance that may host management components. Preconfigure ACL and security group rules to allow communication for basic management componets (NFS - ports XXX, NTP - ports XXX, DNS - ports XXX)|Modify number of virtual server instances and allowed ports in preset or perform the modifications manually|
+|* ensure financial services compliancy for VPC services  \n * perform network setup of all created services  \n * perform network isolation of all created services  \n * ensure all created services are inteconnected with each other|Secure landing zone components|Create a minimum set of required components for a secure landing zone|Create a modified set of required components for a secure landing zone in preset|
+{: caption="Table 1. VPC architecture decisions" caption-side="bottom"}
+
+### PowerVS workspace architecture decisions
+{: #pvs-components}
+
+| Requirement | Component | Choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+|* connect PowerVS workspace with VPC services|Cloud connections|Setup two redundant cloud connections|                    |
+|* configure network for management of all the instances  \n * troughput and latency are not relevant|Management network|Configure private network with default configurations and attch it to both cloud connections|                    |
+|* configure separate network for backup purposes with higher data throughput|Backup network|Configure separate private network with default configurations and attch it to both cloud connections. Networks characteristics might be adapted by the users manually (e.g., to improve throughput)|                    |
+|* preload OS images relevant for customer workload|Pre-loaded OS images|Preload Linux OS images for SAP workload. Keep the number of preloaded images at minimum to save corsts.|Modify the input parameter that specify the list of preloaded OS images.|
+|* preload public SSH key that will be injected into every OS deployment|Pre-loaded SSH public key|Preload customer specified SSH public key|                    |
```

Comment:
```suggestion
|* Connect PowerVS workspace with VPC services|Cloud connections|Set up two redundant cloud connections| |
|* Configure the network for management of all instances  \n * Throughput and latency are not relevant|Management network|Configure private network with default configurations and attach it to both cloud connections| |
|* Configure separate network for backup purposes with higher data throughput|Backup network|Configure separate private network with default configurations and attach it to both cloud connections. Networks characteristics might be adapted by the users manually (for example to improve throughput)| |
|* Pre-load OS images relevant for customer workload|Pre-loaded OS images|Pre-load Linux OS images for SAP workload. Keep the number of pre-loaded images at minimum to save costs.|Modify the input parameter that specifies the list of pre-loaded OS images.|
|* Pre-load a public SSH key that will be injected into every OS deployment|Pre-loaded SSH public key|Pre-load customer specified SSH public key| |
```

- Fix typos
- Make `pre-load` consistent (prefer `preload` or `install` but assume some of this is coming from other sources.
- replace Latin expression `e.g.`